exporter/chronicleexporter/marshal.go

```diff
@@ -20,16 +20,21 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/google/uuid"
+	"github.com/observiq/bindplane-agent/exporter/chronicleexporter/protos/generated"
 	"github.com/observiq/bindplane-agent/expr"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/ottllog"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/plog"
 	"go.uber.org/zap"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 const logTypeField = `attributes["log_type"]`
 
+const collectorID = "aaaa1111-aaaa-1111-aaaa-1111aaaa1111"
```

Comment:
Can we add a comment on these constants? It might not be clear to someone why we're hard-coding the collector ID.

Reply:
Added